cmd/deploy/cfghandler.go

```diff
@@ -81,6 +82,18 @@ func (ch *defaultConfigMapHandler) updateConfigMap(ctx context.Context, cfg *api
 	return errMain
 }
 
+func (h *defaultConfigMapHandler) updateEnvsFromCommands(ctx context.Context, name, namespace string, envs []string) error {
```

Comment:
can we add a comment on what does this func do?

Reply:
done b811281808fb903ba71b9d926f73e713a2141479